tests/sentry/workflow_engine/handlers/action/notification/test_metric_alert.py

```diff
@@ -0,0 +1,411 @@
+import uuid
```

Comment:
🤔 should this live in the `incidents` directory?

Thinking like "when we have this kind of detector, we need to be able to handle this kind of notification".

My thought process around where stuff should live in this example would be "If i delete metric alerts, would i still need this code in the workflow engine?" if the answer is yes, then i feel like it should live in the workflow engine. if the answer is no, then it should likely live in the `incidents` directory.